app/views/admin/emails/index.html.haml

```diff
@@ -20,67 +20,84 @@
                 %a.btn.btn-link.control_label.template_help_link{"data-name"=>"updated_registrations_dates_help"} Show Help
                 = render partial: 'help', locals: {id: 'updated_registrations_dates_help', show_event_variables: false}
                 = f.input :send_on_venue_updated, label: 'Send an email on updating the Venue.', input_html: {"data-name"=>"email_settings_venue_updated_subject", "class"=>"send_on_radio"}
                 = f.input :venue_updated_subject
                 = f.input :venue_updated_body, input_html: { rows: 10, cols: 20 }
-                %a.btn.btn-link.control_label.load_template{"data-template"=>"Dear {name},\n\nThe Conference venue of {conference} has changed. New location is: {venue}.\n Address: {venue_address}.\n\nFeel free to contact us with any questions or concerns.\n\nWe look forward to seeing you there.\n\nBest wishes\n\n{conference} Team",
-                "data-name"=>"email_settings_venue_updated_body"} Load Template
+                %a.btn.btn-link.control_label.load_template{'data-subject-input-id' => 'email_settings_venue_updated_subject',
+                'data-subject-text' => 'The venue has changed',
+                'data-body-input-id'=>'email_settings_venue_updated_body',
+                'data-body-text'=>"Dear {name},\n\nThe Conference venue of {conference} has changed. New location is: {venue}.\n Address: {venue_address}.\n\nFeel free to contact us with any questions or concerns.\n\nWe look forward to seeing you there.\n\nBest wishes\n\n{conference} Team"} Load Template
                 %a.btn.btn-link.control_label.template_help_link{"data-name"=>"updated_venue_help"} Show Help
                 = render partial: 'help', locals: {id: 'updated_venue_help', show_event_variables: false}
               #cfp.tab-pane{role: "tabpanel"}
                 = f.input :send_on_program_schedule_public, hint: "This will notify all participants when the dates are updated or when the schedule is made public"
                 = f.input :program_schedule_public_subject, hint: "This subject will used whenever dates are updated or when the schedule is made public"
                 = f.input :program_schedule_public_body, input_html: { rows: 10, cols: 20 }
-                %a.btn.btn-link.control_label.load_template{"data-template"=>"Dear {name},\n\nThe Conference Call for Papers Details of {conference} has changed.The schedule is being made public.\n Link to Schedule {schedule_link} \n\nBest wishes\n\n{conference} Team",
-                "data-name"=>"email_settings_program_schedule_public_body"} Load Template
+                %a.btn.btn-link.control_label.load_template{'data-subject-input-id' => 'email_settings_program_schedule_public_subject',
+                'data-subject-text' => 'The schedule has been released',
```

Comment:
s/released/announced

We should probably rewrite the body of that template to something like:
```
Dear {name},

The schedule for {conference} has been announced.
Link to Schedule {schedule_link} 

Best wishes
{conference} Team
```